cmstestsuite/TestCleanCheckout.py

```diff
@@ -77,7 +77,8 @@ def setup_cms():
     os.environ["PYTHONPATH"] = "%(TEST_DIR)s" % CONFIG
 
     info("Building cms.")
-    sh("./setup.py build")
+    sh("./prerequisites.py compile_l10n")
+    sh("./prerequisites.py compile_isolate")
```

Comment:
Yes, that's exactly what I meant. Anyway, I understand that your idea is to use setup.py only for Python package-related stuff, which is fine. To me setup was a better name than prerequisite but it's not too important.

Why do you say you'd have 3 commands instead of 2?